ansible_runner/runner_config.py

```diff
@@ -145,6 +145,8 @@ def __init__(self,
         self.ssh_key_data = ssh_key
         self.execution_mode = ExecutionMode.NONE
         self.envvars = envvars
+        # for strict container isolation, keep a rolling list of what env vars should be exposed to job
+        self.env_var_whitelist = ['PROJECT_UPDATE_ID']
```

Comment:
lets not use `whitelist` here. In the ansible project itself, we've made a conscious push towards `allowlist`